### PR TITLE
Design changes

### PR DESCRIPTION
**Pull request by kistora**
### Description of your PR

Change general app design by using [Angular material theming doc](https://material.angular.io/guide/theming).

### Pre-review TODO

- [x] PR is rebased on top of `origin/master` 
- [x] PR provide new tests for new behaviors
- [ ] PR doesn't introduce commented out code
- [x] PR has been tested locally on the main use cases
- [ ] Documentation have been updated if necessary

### CODE REVIEW

File: client/src/app/footer/footer.component.scss [hunk 1]
```diff
@@ -17,3 +17,7 @@
 a {
   color: rgba(0, 0, 0, .87);
 }
+
+.accent {
+  color: #F44336; // accent color
```
[the remark this answers is not in the document]
Comment (kistora) [reply client/src/app/footer/footer.component.scss]: Yes :)

File: client/src/app/menu/menu.component.html [hunk 2]
```diff
@@ -43,6 +43,6 @@ <h3 mat-subheader *ngIf="subMenu.name">{{subMenu.name}}</h3>
 	</mat-sidenav>
 	<mat-sidenav-content [style.marginTop.px]="mobileQuery.matches ? 56 : 64" class="content-container">
 		<router-outlet></router-outlet>
-		<div style="height: 128px;"></div>
+		<div style="height: 180px;"></div>
```
[the remark this answers is not in the document]
Comment (kistora) [reply client/src/app/menu/menu.component.html]: Yep

File: client/src/app/menu/menu.component.scss [hunk 3]
```diff
@@ -29,3 +29,8 @@
 .connectedUserInformation {
   padding: 0 20px 0 20px;
 }
+
+.header {
+  font-family: "MedievalSharp", cursive;
```
[the remark this answers is not in the document]
Comment (kistora) [reply client/src/app/menu/menu.component.scss]: Yes!

[the remark this answers is not in the document]
Comment (kistora) [reply client/src/app/menu/menu.component.scss]: Yes

File: client/src/index.html [hunk 4]
```diff
@@ -18,6 +18,53 @@
...
+    animation: dots 1.5s ease-out infinite;
+  }
+  .d-2 {
+    animation-delay: .5s;
+  }
+  .d-3 {
+    animation-delay: 1s;
+  }
+  </style>
+
```
[the remark this answers is not in the document]
Comment (kistora) [reply client/src/index.html]: No we can't ! More information: https://alligator.io/angular/custom-loading-screen/

File: client/src/styles.scss [hunk 5]
```diff
@@ -1,8 +1,11 @@
-// Angular material
-@import "~@angular/material/prebuilt-themes/deeppurple-amber.css";
 // Picker
 @import "~ng-pick-datetime/assets/style/picker.min.css";
 
+<<<<<<< HEAD
+=======
+@import url("https://fonts.googleapis.com/css?family=MedievalSharp");
+
+>>>>>>> Fix lint errors
```
[the remark this answers is not in the document]
Comment (kistora) [reply client/src/styles.scss]: Yes we can but i prefere this way :)

[the remark this answers is not in the document]
Comment (Embraser01) [reply client/src/styles.scss]: Actually, we already import Roboto through html like proposed by google font. And we could import it in only one http call:
`<link href="https://fonts.googleapis.com/css?family=MedievalSharp|Roboto:300,400,500" rel="stylesheet">`

Anyway I don't really mind if it's imported through HTML or through CSS, but I think we have to be consistent :wink:

[the remark this answers is not in the document]
Comment (kistora) [reply client/src/styles.scss]: Yep I've seen that :) Let's do it with the CSS import way :)

**Pull request merged**